test/fixtures/server.js

```diff
@@ -647,6 +647,26 @@ function createDigestServer (options) {
     return app;
 }
 
+function createResponseErrorServer () {
```

Comment:
This is not required.

Reply:
How are we going to test for response error without creating a server?

Reply:
Request or response error doesn't matter here.
We just need to make sure the history object is preset on error.